pkg/clients/eks/eks.go

```diff
@@ -302,8 +302,13 @@ func IsUpToDate(p *v1beta1.ClusterParameters, cluster *eks.Cluster) (bool, error
 		}
 	}
 
+	// if a, r := awsclients.DiffTags(p.Tags, cluster.Tags); len(a) != 0 || len(r) != 0 {
+	// 	return false, nil
+	// }
+
 	return cmp.Equal(&v1beta1.ClusterParameters{}, patch, cmpopts.EquateEmpty(),
 		cmpopts.IgnoreTypes(&v1alpha1.Reference{}, &v1alpha1.Selector{}),
+		// cmpopts.IgnoreFields(v1beta1.ClusterParameters{}, "Tags"),
```

Comment:
Seems like these additions are non-functional.

Reply:
Thanks for catching this! Forgot to remove them from earlier testing.

Reply:
Fixed :+1: